package.json

```diff
@@ -29,19 +32,16 @@
     "pg": "^7.11.0",
     "pg-hstore": "^2.3.2",
     "sequelize": "^5.8.6",
-    "sequelize-cli": "^5.4.0",
+    "sequelize-cli": "^5.5.0",
     "slug": "^1.1.0",
     "uuid": "^3.3.2",
     "validatorjs": "^3.15.1"
   },
   "devDependencies": {
-    "@babel/plugin-transform-runtime": "^7.4.4",
-    "@babel/preset-env": "^7.4.5",
-    "@babel/register": "^7.4.4",
-    "babel-plugin-module-resolver": "^3.2.0",
     "chai": "^4.2.0",
     "chai-as-promised": "^7.1.1",
     "chai-http": "^4.3.0",
+    "chai-uuid": "^1.0.6",
```

Comment:
Why are you installing this? Are you manually testing uuids?

Reply:
the validation test with regex rules but 'chai-uuid' test the actual uuids.

Reply:
Where are you testing uuid point me to a line...

Reply:
I just did with a comment